MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+include CHANGELOG.rst
+include LICENSE
+include README.rst
+
+recursive-include tests *
```

Comment:
Fixed in https://github.com/Kinto/kinto-webpush/commit/dcab26e1302377b9ea5caf8c431936cc6cab0c0d